requirements.txt

```diff
@@ -10,3 +10,4 @@ jax==0.2.9
 jaxlib==0.1.59
 hilbertcurve==1.0.5
 networkx==2.5
+cvxpylayers==0.1.5
```

Comment:
note to self to ensure that whatever future version of jax we upgrade to is compatible with this version of cvxpylayers...

Reply:
That would make a good comment actually

Reply:
Addressed in https://github.com/proteneer/timemachine/pull/399/commits/6afd987a2cb955d5e773b3a59303de425a6210a8